src/Extension/ProductVariationInventoryManager.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Dynamic\Foxy\Inventory\Extension;
+
+use Dynamic\Foxy\Orders\Model\OrderDetail;
+use Dynamic\Foxy\Orders\Model\OrderVariation;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\ORM\DataList;
+
+class ProductVariationInventoryManager extends ProductInventoryManager
+{
+    /**
+     * @return DataList
+     */
+    public function getOrders()
```

Comment:
Seems like an opportunity to make this more efficient

Reply:
Same as other inventory manager class.